include/exiv2/value.hpp

```diff
@@ -1274,6 +1298,61 @@ namespace Exiv2 {
         ValueList value_;
 
     private:
+        //! Utility for toInt64, toUint32, etc.
+        template<typename I>
+        inline I float_to_integer_helper(long n) const {
```

Comment:
According to codecov, many of these new lines are not covered by tests. Did you check if this new code is covered? I know that codecov sometimes report false issues.